1/test/Todos.spec.js

```diff
@@ -0,0 +1,10 @@
+import {mount} from "@vue/test-utils"
+import todos from '@/components/Todos.vue'
```

Comment:
Use PascalCase for components.
```suggestion
import Todos from '@/components/Todos.vue'
```